src/phpDocumentor/Compiler/Pass/Debug.php

```diff
@@ -48,11 +48,11 @@ public function getDescription(): string
         return 'Analyze results and write report to log';
     }
 
-    public function __invoke(ProjectInterface $project): ProjectInterface
+    public function __invoke(DocumentationSetDescriptor $documentationSet): DocumentationSetDescriptor
     {
-        $this->analyzer->analyze($project);
+        $this->analyzer->analyze($documentationSet);
```

Comment:
This should only run on apiSet?

Reply:
The analyzer itself does varying things depending on the type of documentation set